src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/AdvancedSettingsModal.js

```diff
@@ -0,0 +1,93 @@
+import { Field, Help, TextInput } from '@1hive/1hive-ui'
+import React, { Fragment, useCallback, useEffect, useState } from 'react'
+import { Modal, PercentageField } from '@components/Onboarding//kit'
+
+const AdvancedSettingsModal = ({
+  requestTokenAddress,
+  minThresholdStakePct,
+  stakeOnProposalPct,
+  visible,
+  onClose,
+  onDone,
+}) => {
+  const [requestToken, setRequestToken] = useState(requestTokenAddress)
+  const [currMinThresholdStakePct, setCurrMinThresholdStakePct] = useState(
+    minThresholdStakePct
+  )
+
+  const handleMinThresholdChange = useCallback(
+    value => {
+      setCurrMinThresholdStakePct(value)
+    },
+    [setCurrMinThresholdStakePct]
+  )
+
+  const handleRequestTokenChange = useCallback(
+    event => {
+      setRequestToken(event.target.value)
+    },
+    [setRequestToken]
+  )
+
+  const handleDone = () => {
+    onDone({ currMinThresholdStakePct, requestToken })
+    onClose()
+  }
+
+  useEffect(() => {
+    setCurrMinThresholdStakePct(minThresholdStakePct)
+    setRequestToken(requestTokenAddress)
+  }, [minThresholdStakePct, requestTokenAddress])
+
+  return (
+    <Modal
+      title="Advanced Settings"
+      visible={visible}
+      onClick={handleDone}
+      onClose={onClose}
+      width="100"
```

Comment:
Ah i think we need to add the unit here.
If we fix here we can remove the width set to the `Field`

```suggestion
      width="600px"
```